test/support/factory.ex

```diff
@@ -19,6 +19,14 @@ defmodule Wanda.Factory do
 
   alias Wanda.Results.ExecutionResult
 
+  def map_factory(attrs) do
```

Comment:
I could very well name this `env_factory` to be more specific, as it is only used to build the `env` data